spec/api_spec.rb

```diff
@@ -20,30 +24,57 @@
   end
 
   describe "POST /interviewer" do
-    let(:database) {{'interviewer' => collection}}
-    let(:interviewer) {{name:'Jichao', email:'jichao@thoughtworks.com'}}
+
     before do
-      allow(collection).to receive(:set) {double(value: interviewer)}
+      allow(collection).to receive(:create) { double(value: interviewer) }
     end
     it "returns new interviewer id" do
-
       post "/api/v1/interviewer", interviewer
       expect_json(interviewer)
     end
   end
 
   describe "GET /interviewers" do
-    let(:database) {{'interviewer' => collection}}
-    let(:interviewer1) {{name:'Jichao', email:'jichao@thoughtworks.com'}}
-    let(:interviewer2) {{name:'Ouyang', email:'ouayng@thoughtworks.com'}}
-    before do
-      expect(collection).to receive(:find_all).and_return([double(value: interviewer1),double(value: interviewer2) ])
-    end
-    it "returns all interviewers" do
+    let(:collection) { [double(value: interviewer1, key: '001'), double(value: interviewer2, key: '002')] }
+    let(:interviewer1) { {name: 'Jichao', email: 'jichao@thoughtworks.com', key: '001'} }
+    let(:interviewer2) { {name: 'Ouyang', email: 'ouayng@thoughtworks.com', key: '002'} }
 
+    it "returns all interviewers" do
       get "/api/v1/interviewers"
-      expect(json_body).to eql([interviewer1,interviewer2])
+      expect(json_body).to eql([interviewer1, interviewer2])
+    end
+  end
+
+  describe 'GET /interviewer/:id' do
+    let(:key) { '123456' }
+    let(:collection) { {'123456' => interviewer_record_in_db} }
+    it "returns interviewer with key=:id" do
+      get "/api/v1/interviewer/#{key}"
+      expect(json_body).to eql({name: 'Jichao', email: 'jichao@thoughtworks.com', key: '123456'})
```

Comment:
Line is too long. [97/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.